config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
Nemohla by som to spravit ako .rb subor v initializers? Alebo preco je lepsie pouzit yml?